### DISCUSSION TITLE
QLXD Lockup

### DISCUSSION THREAD

**Discussion by drmmr763** [karlcswanson/micboard#28] [Q&A] [answered]
Hello

I just purchased some QLXD devices and I was a little excited to use Micboard finally. Unfortunately, I see there's a known firmware issue. 

1. Does anyone know if this is still an active issue with more recent firmware? The was a firmware release as recently as Dec 2021.
2. The docs describe the device as "locking up". What does that mean exactly? Does it still operate the microphones even though the front panel is locked? Does a reboot fix it?

**Comment by karlcswanson** [comment, marked as answer]
Hello @drmmr763,
I submitted the bug to Shure in 2018 and they replicated and acknowledged the issue.  I tested the next couple of firmware updates for a fix, but they had not fixed the bug.  I have not been able to check recent updates as I don't currently have access to QLX-D devices.

The test instructions are simple if you would like to test current firmware.  It's been a while, but I think the receiver still passed audio, but once it locked up, it was inaccessible via WWB and the front panel.  Production devices should not lock up if another computer on the network gets unplugged.

https://github.com/karlcswanson/micboard/blob/master/docs/qlxd.md

**Comment by s0600204** [comment]
Just as an update: about a month ago (June 2025) I acquired a QLXD4 receiver, and after testing can state that the problem appears to have been fixed.

Reading the release notes of each firmware release it is unclear *when* it was resolved, but neither the earliest nor the most recent of the three firmware versions currently available via SUU (`2.5.2` and `2.7.1`) appear to have the problem. (I didn't test the `2.6.2` version.)

(If I may, I'd also like to point out that the suggested workaround - of rolling back to `2.1.5` - is no longer (easily) possible, due to that version not being available for download (via SUU) from Shure's website.)
